### DISCUSSION TITLE
Support for Cookies

### DISCUSSION THREAD

**Discussion by Purvik** [lejard-h/chopper#366] [Q&A] [answered]
We're having Chopper setup for networking layer with interceptors and we want to have configured cookies with it for APIs. Login will respond with cookies which we need to send back for all subsequent request. Is there any way we can achieve this in chopper? 
For reference, we refer https://pub.dev/packages/cookie_jar which is having support for http and dio package.

**Comment by techouse** [comment, marked as answer]
Hmm... Can't the [`Authenticator`](https://github.com/lejard-h/chopper/blob/master/chopper/lib/src/authenticator.dart) class handle this?

Check the [FAQ for an example](https://github.com/lejard-h/chopper/blob/develop/faq.md#authorized-http-requests-using-the-special-authenticator-interceptor).
